Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
+# dev dependency
+gem 'pedicel-pay', git: 'https://github.com/cub8/pedicel-pay', branch: 'openssl-30-support'
```

Comment:
Temporary dependency, because pedicel-pay doesn't work with OpenSSL 3.0.

Once [this PR](https://github.com/clearhaus/pedicel-pay/pull/16) is accepted it should be removed.